src/AvaTaxError.cs

```diff
@@ -10,13 +10,19 @@ public class AvaTaxError : Exception
         /// <summary>
         /// The raw error message from the client
         /// </summary>
-        public ErrorResult error { get; set; }
+        public ErrorResult error { get; private set; }
```

Comment:
Let's keep this out of the logging PR to keep it focused on one single feature at a time.